README.md

```diff
@@ -1,34 +1,156 @@
 o-editorial-typography [![Circle CI](https://circleci.com/gh/Financial-Times/o-editorial-typography/tree/master.svg?style=svg)](https://circleci.com/gh/Financial-Times/o-editorial-typography/tree/master)[![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
 =================
 
-**Work in progress: See the [o-typography v6 proposal](https://github.com/Financial-Times/o-typography/issues/203).**
-
-_Typography styles for editorial content _
+_Typography styles for editorial content._
 
 - [Markup](#markup)
 - [Sass](#sass)
 - [Contact](#contact)
 - [Licence](#licence)
 
-_Whatever usage instructions your component has. We've broken this down by Markup, JavaScript and Sass, but it depends how complex your component is._
+[o-typography](https://registry.origami.ft.com/components/o-typography) provides fundamental typographic tooling and styles for user interfaces. o-editorial-typography builds on that with typography for editorial content including headlines, tags, and bylines. It makes no assumption about where the editorial typography will be used -- it could be a page, another component, or email for example -- so no default margins are provided. To apply typography and layout editorial typography within an article see [o-editorial-layout](https://registry.origami.ft.com/components/o-editorial-layout).
 
 ### Markup
 
-_Common templating can go here, especially if there is only one template, but people can always check the demos for more._
+#### Headings
+
+Headings 1-6 are provided with the class `o-editorial-typography-heading-level-[level]`:
+
+```html
+<h1 class="o-editorial-typography-heading-level-1">Heading level 1</h1>
+<h2 class="o-editorial-typography-heading-level-2">Heading level 2</h2>
+<h3 class="o-editorial-typography-heading-level-3">Heading level 3</h3>
+<h4 class="o-editorial-typography-heading-level-4">Heading level 4</h4>
+<h5 class="o-editorial-typography-heading-level-5">Heading level 5</h5>
```

Comment:
I think we should to finish off the list with the h6 here, as it is the only one missing in the example
```suggestion
<h5 class="o-editorial-typography-heading-level-5">Heading level 5</h5>
<h6 class="o-editorial-typography-heading-level-6">Heading level 6</h6>
```